.gitmodules

```diff
@@ -9,23 +9,23 @@
 	url = https://github.com/cuberite/ChatLog.git
 [submodule "lib/mbedtls"]
 	path = lib/mbedtls
-	url = https://github.com/cuberite/polarssl.git
```

Comment:
Don't change the submodule URLs, it breaks things! Just run a git pull in our vendored version if necessary.